ignite/contrib/metrics/precision_recall_curve.py

```diff
@@ -73,3 +75,32 @@ def __init__(self, output_transform: Callable = lambda x: x, check_compute_fn: b
         super(PrecisionRecallCurve, self).__init__(
             precision_recall_curve_compute_fn, output_transform=output_transform, check_compute_fn=check_compute_fn
         )
+
+    def compute(self) -> Tuple[torch.Tensor, torch.Tensor, torch.Tensor]:
```

Comment:
I think returned value type is float.

Reply:
But I have casted to torch.Tensor in line 102, I have done that as the input array is also torch.Tensor.

Reply:
```suggestion
    def compute(self) -> Tuple[float, float, float]:
```

Reply:
Ok i realised that the values are array type.